packages/teleport-component-generator-react/src/react-mapping.ts

```diff
@@ -21,6 +21,14 @@ export const ReactMapping: Mapping = {
         version: '2.2.1',
       },
     },
+    'markdown-node': {
+      elementType: 'Markdown',
```

Comment:
```js
elementType: 'markdown'
```

Please 😀

Reply:
Should we build something like `dangerous-html` for this use-case too ?

Reply:
This is how the element will look in jsx code, why with small letter? No need to build something new, this one works well.

Reply:
for now we're focusing on publishing with next.js the things related to cms, if we will intend to support other frameworks we will indeed need to find another way.

Reply:
yeah sure sure, no problem 👍